cmd/root.go

```diff
@@ -66,6 +73,17 @@ Furyctl is a simple CLI tool to:
 				analytics.Version(versions["version"])
 				analytics.Disable(cobrax.Flag[bool](cmd, "disable-analytics").(bool))
 			},
+			PersistentPostRun: func(cmd *cobra.Command, _ []string) {
+				// Show update message if available at the end of the command
+				select {
+				case release := <-r:
+					if release.Version != versions["version"] {
```

Comment:
nitpick because it won't be happening in a normal use case but for example with versions: 0.10.0 (branch) and 0.8.0 (GitHub latest tag) it still says:
```
INFO[0000] New furyctl version available: 0.10.0 => v0.8.0
```